sklearn/metrics/classification.py

```diff
@@ -934,7 +925,7 @@ def precision_recall_fscore_support(y_true, y_pred, beta=1.0, labels=None,
         labels = np.hstack([labels, np.setdiff1d(present_labels, labels,
                                                  assume_unique=True)])
 
-    ### Calculate tp_sum, pred_sum, true_sum ###
+    # Calculate tp_sum, pred_sum, true_sum ###
```

Comment:
Why have you changed these? the number of hashes on left and right had been equal to indicate sections rather than ordinary comments. Don't touch more than you need to, in general!